configs/frontend_config/patient-summary.json

```diff
@@ -0,0 +1,171 @@
+{
+  "@openmrs/esm-patient-chart-app": {
+    "extensionSlots": {
+      "hiv-patient-summary-dashboard": {
+        "add": [
+          "encounter-tiles-group"
+        ],
+        "configure": {
+          "encounter-tiles-group": {
+            "tilesDefinitions": [
+              {
+                "columns": [
```

Comment:
@hadijahkyampeire for each `tileDefinition`, we need to add a `tileHeader` which should be translatable.
That way the component won't need to know about how many tiles to deal with. It will just loop through the tiles and render each with it's header and corresponding columns

Reply:
okay, I will add that.